sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -995,6 +993,40 @@ public void testPrimitiveEarliestInSubquery() throws Exception
     );
   }
 
+  @Test
+  public void testPrimitiveEarliestInSubqueryGroupBy() throws Exception
```

Comment:
This doesn't look like a subquery that is being tested

Reply:
updated